### PR TITLE
make sure all config done with the installer is represented in the emergency backup template as well

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

File: htconfig.php [hunk 1]
```diff
@@ -60,9 +64,16 @@
 
 $a->config['system']['huburl'] = '[internal]';
 
+// Server-to-server private message encryption (RINO) is allowed by default.
+// Encryption will only be provided if this setting is true and the
```
Comment (annando) [inline htconfig.php]: "setting is true" sounds confusing for a non boolean value ;-)

Comment (tobiasd) [inline htconfig.php]: right :-) that comment is still from the 0/1 time

**Comment by zotlabs** [comment]
FYI, mcrypt is not available at all in PHP 7.1. I know mcrypt is hardly used for RINO anymore, but it's needed for PHPSecLib and Diaspora iirc. Diaspora is easy to replace with openssl but it's likely you'll need to upgrade phpseclib and completely remove RINO1 compatibility.

**Comment by annando** [comment]
@zotlabs thanks for the hint! Sounds like a task for 3.5.2

**Comment by annando** [comment]
@fabrixxm this crypto stuff sounds like a job for you ;-)

**Pull request merged**